internal/cmd/grants.go

```diff
@@ -132,6 +83,86 @@ func newGrantsListCmd(cli *CLI) *cobra.Command {
 	return cmd
 }
 
+func userGrants(cli *CLI, client *api.Client, grants *api.ListResponse[api.Grant]) (int, error) {
```

Comment:
would be nice to use generics here since the code is _very_ similar between this and `groupGrants` but the table headers are different so not sure how to genericize it